flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/async/AsyncWaitOperator.java

```diff
@@ -276,10 +232,10 @@ public void snapshotState(StateSnapshotContext context) throws Exception {
 	@Override
 	public void initializeState(StateInitializationContext context) throws Exception {
 		super.initializeState(context);
+
```

Comment:
nit: unnecessary change. If we want to reformat the codes it is better for a separate hotfix commit.